cli/command/system/prune.go

```diff
@@ -0,0 +1,90 @@
+package system
+
+import (
+	"fmt"
+
+	"github.com/docker/docker/cli"
+	"github.com/docker/docker/cli/command"
+	"github.com/docker/docker/cli/command/prune"
+	units "github.com/docker/go-units"
+	"github.com/spf13/cobra"
+)
+
+type pruneOptions struct {
+	force bool
+	all   bool
+}
+
+// NewPruneCommand creates a new cobra.Command for `docker du`
+func NewPruneCommand(dockerCli *command.DockerCli) *cobra.Command {
+	var opts pruneOptions
+
+	cmd := &cobra.Command{
+		Use:   "prune [COMMAND]",
+		Short: "Remove unused data",
+		Args:  cli.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runPrune(dockerCli, opts)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.BoolVarP(&opts.force, "force", "f", false, "Do not prompt for confirmation")
+	flags.BoolVarP(&opts.all, "all", "a", false, "Remove all unused images not just dangling ones")
+
+	containerPruneCmd := prune.NewContainerPruneCommand(dockerCli)
+	containerPruneCmd.Use = "containers"
+	volumePruneCmd := prune.NewVolumePruneCommand(dockerCli)
+	volumePruneCmd.Use = "volumes"
+	imagePruneCmd := prune.NewImagePruneCommand(dockerCli)
+	imagePruneCmd.Use = "images"
+
+	cmd.AddCommand(
+		containerPruneCmd,
+		volumePruneCmd,
+		imagePruneCmd,
+	)
```

Comment:
In the end, I removed it for now. As @thaJeztah always say it's easier to add later than remove :)